packages/oui-navbar/_mixins.less

```diff
@@ -6,136 +6,461 @@
 
 #oui {
   .navbar(
-    @select-selector,
-    @background-color: @oui-navbar-background-color,
+    @font-size: @oui-navbar-font-size,
+    @height: @oui-navbar-height,
+    @padding: @oui-navbar-padding,
     @font-color: @oui-navbar-font-color,
-    @icon-color: @oui-navbar-icon-color
+    @font-color_active: @oui-navbar-font-color_active,
+    @background-color: @oui-navbar-background-color,
+    @background-color_hover: @oui-navbar-background-color_hover,
+    @z-index: @oui-navbar-z-index,
+    @brand-color: @oui-navbar-brand-color,
+    @brand-padding: @oui-navbar-brand-padding,
+    @backdrop-opacity: @oui-navbar-backdrop-opacity
   ) {
     display: flex;
+    flex-direction: row;
     flex-wrap: wrap;
     align-items: center;
-    justify-content: space-between;
-    padding: rem-calc(8) rem-calc(16);
-    background-color: @background-color;
-    color: @font-color;
-    white-space: nowrap;
+    justify-content: flex-start;
+
+    padding: @padding;
+    position: relative;
     user-select: none;
+    font-size: @font-size;
+    color: @font-color;
+    background-color: @background-color;
+    z-index: @z-index;
 
-    .oui-icon.oui-icon-ovh {
-      color: @icon-color;
-      font-size: rem-calc(30);
-      vertical-align: middle;
+    // variant 'fixed'
+    &_fixed {
+      position: fixed;
+      top:0;
+      left: 0;
+      right: 0;
     }
 
-    &__title,
-    &__title-link,
-    &__item,
-    &__link {
+    // Header/Brand
+    &__brand {
+      height: @height;
+      line-height: @height;
+      display: block;
+      padding: @brand-padding;
+
       &,
-      &:active,
-      &:focus,
       &:hover,
-      &:visited,
-      & a:link {
-        color: @font-color;
+      &:focus {
         text-decoration: none;
+        color: @brand-color;
+      }
+
+      .oui-icon {
+        vertical-align: initial;
+        font-size: rem-calc(44);
+        color: inherit;
       }
     }
+  }
 
-    &__title {
-      display: inline-block;
-      margin: 0;
-      color: @font-color;
+  .navbar-backdrop(
+    @height: @oui-navbar-height,
+    @z-index: @oui-navbar-z-index,
+    @backdrop-opacity: @oui-navbar-backdrop-opacity
+  ) {
+    position: fixed;
+    top: @height;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    background: black;
+    transition: opacity 0.25s ease-out;
+    pointer-events: none;
+    z-index: @z-index;
+
+    @media screen and (max-width: 1024px) {
```

Comment:
We have responsive mixins to handle this. If not, is it possible to redefine another mixins and therefore avoid redefining break-points?

Reply:
Yes, i'll look for them